robot/basestation/stream_capture.py

```diff
@@ -72,6 +77,59 @@ def stop_recording_feed(stream_url):
     print(message)
     return success, message
 
+def add_rotation(stream_url,rotation):
+    """
+    Add rotation value to the value of the key (stream_url) in active recordings
+    Converts the list to a tuple and handles errors
+    """
+    global active_recordings
```

Comment:
just to follow up on the previous comment about the use of global, this here is an example of the proper way